commanderbot_ext/faq/faq_guild_state.py

```diff
@@ -82,26 +94,25 @@ async def update_faq(
         else:
             await ctx.send(f"No FAQ named `{faq_name}`")
 
-    async def add_alias(self, ctx: Context, faq_name: str, faq_alias: str):
+    async def add_alias(self, ctx: Context, faq_name: str, faq_alias: str, guild: Guild):
```

Comment:
`guild` isn't needed as a param here since we already have `self.guild` as part of `GuildState`.